samples/highcharts/demo/polar-advanced/demo.js

```diff
@@ -0,0 +1,479 @@
+const
+    monthExtremes = { min: 0, max: 26 },
+    weekExtremes = { min: 1, max: 5 },
+    paneOpeningAngles = { startAngle: 40.5, endAngle: 319.5 },
+    noLabelProp = { labels: { enabled: false } },
+    specialSeriesProps = {
+        showInLegend: false,
+        groupPadding: 0,
+        pointPadding: 0
+    },
+    toggleableGradient = {
+        pattern: undefined,
+        radialGradient: [1, 0.25, 0.1],
+        stops: [
+            [0, '#1f1836'],
+            [1, '#45445d']
+        ]
+    },
+    mouseOut = function () {
+        const chart = this.series.chart;
+        chart.setMidPaneBg({
+            backgroundColor: toggleableGradient,
+            outerRadius: '75%'
+        });
+        chart.subtitle.element.style.opacity = 1;
+    },
+    colors = Highcharts.getOptions().colors,
+    asHcColor = hexval => new Highcharts.Color(hexval),
+    // Declaring colors used more than once
+    hcColor0 = asHcColor(colors[0]),
+    hcColor1 = asHcColor(colors[1]),
+    hcColor5 = asHcColor(colors[5]),
+    hcColor8 = asHcColor(colors[8]),
+    hcColor9 = asHcColor(colors[9]),
```

Comment:
This is kind of subjective, but instead of a list of named constants with numbers in them, in this case I would just do a map into a modified array.

```js
colors = Highcharts.getOptions().colors.map(color => new Highcharts.Color(color));
```

Or, if I am not mistaken, we can shorten it using `Color.parse`:

```js
colors = Highcharts.getOptions().colors.map(Highcharts.Color.parse);
```

Reply:
I agree with your preference here, but, if i recall correctly, I had problems with the constructor when putting them all in an array using the "map" function.

I will try your second suggestion!

Reply:
It did work!